external/kinesis-asl/src/main/python/examples/streaming/kinesis_wordcount_asl.py

```diff
@@ -34,10 +35,9 @@
       $ export AWS_SECRET_KEY=<your-secret-key>
 
       # run the example
-      $ bin/spark-submit -jars external/kinesis-asl/target/scala-*/\
-        spark-streaming-kinesis-asl-assembly_*.jar \
+      $ bin/spark-submit --jars 'external/kinesis-asl-assembly/target/spark-streaming-kinesis-asl-assembly_*.jar' \
```

Comment:
Thank you for your contribution, @sekikn . Could you fix the Python style violation?
```
./external/kinesis-asl/src/main/python/examples/streaming/kinesis_wordcount_asl.py:38:101: E501 line too long (115 > 100 characters)
[error] running /home/jenkins/workspace/SparkPullRequestBuilder/dev/lint-python ; received return code 1
```

Reply:
Thanks for the review @dongjoon-hyun, just updated!